src/assets/TestFramework/RecordedTestBase.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using NUnit.Framework;
+
+namespace Azure.Core.TestFramework
+{
+    [TestFixture(true)] // this is only mocking the RecordedTestBase to make sure the test could properly run and skip
+    public abstract class RecordedTestBase<TEnvironment> where TEnvironment : TestEnvironment, new()
```

Comment:
Is this class similar as `ManagementRecordedTestBase`? in the future we will consolidate as one for both mgmt and dpg?

Reply:
ManagementRecordedTestBase inherits this.